### PR TITLE
.github/workflows: create quick boot mysql images

### PR DESCRIPTION
**Pull request by ronenlu**

### CODE REVIEW

File: .github/workflows/cd-push-docker-mysql.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build & Push ${{ matrix.dialect }} Docker Image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./ops/mysql/Dockerfile
+          push: true
+          tags: arigaio/${{ matrix.dialect }}-${{ github.sha }},arigaio/${{ matrix.dialect }}-latest
```
Comment (giautm) [inline .github/workflows/cd-push-docker-mysql.yaml:28]: I dont think you have a valid image tag here. Double : in its name

**Review by giautm — CHANGES_REQUESTED** [review]

File: .github/workflows/cd-push-docker-mysql.yaml (hunk 1 above)
Comment (ronenlu) [reply .github/workflows/cd-push-docker-mysql.yaml:28]: what do you suggest?
just arigaio/${{ matrix.dialect }} ?

File: ops/mysql/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+ARG DIALECT=mysql:8.0
```
Comment (a8m) [inline ops/mysql/Dockerfile:1]: Move this file under `.github`. There's no need to have it under the top level repository.

File: .github/workflows/cd-push-docker-mysql.yaml (hunk 1 above)
Comment (giautm) [reply .github/workflows/cd-push-docker-mysql.yaml]: We should add a prefix. `arigaio/eph-${{ matrix.dialect }}`. Or we need to separate the dialect and its version?

Comment (a8m) [reply .github/workflows/cd-push-docker-mysql.yaml]: I prefer `arigaio/mysql:<tag>`.

File: ops/mysql/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+FROM $DIALECT as builder
+
+ARG MYSQL_DATABASE=test
+ARG MYSQL_USER=user
+ARG MYSQL_PASSWORD=pass
+ARG MYSQL_ROOT_PASSWORD=pass
+
+ENV MYSQL_ROOT_PASSWORD=$MYSQL_ROOT_PASSWORD MYSQL_DATABASE=$MYSQL_DATABASE MYSQL_USER=$MYSQL_USER MYSQL_PASSWORD=$MYSQL_PASSWORD
+
+# That file does the DB initialization but also runs mysql daemon, by removing the last line it will only init
```
Comment (rotemtam) [inline ops/mysql/Dockerfile:12]:
```suggestion
# Remove the last line of the entry point script, leaving the initialization code but omitting actually starting the db.
```

**Review by a8m — APPROVED** [review]

**Pull request merged**